### ISSUE TITLE
[Feature Request] LibreTube integration > Remote Casting

### ISSUE THREAD

**Issue by SecularSteve** [yuliskov/SmartTube#2818]
Instead of having to configure it manually for the Mobile YouTube App, what if it worked out of the box for LibreTube?

No configurations, no codes, just open the app, choose the option, and play from Libre to Smart.

Any opinions?

**Comment by daddy32** [comment]
Yeah, the requirement to use the official Youtube app for casting is the biggest drawback (for me at least).

**Comment by SecularSteve** [comment]
Originally mentioned by https://github.com/Eco-Gaming

-----

Another option for casting would be using FCast:
https://fcast.org/
https://gitlab.futo.org/videostreaming/fcast

It's a source available MIT-licenced chromecast alternative. FCast is used in FUTOs client for Youtube, etc. (GrayJay (GitLab)) and according to the devs it's supposed to be easy to implement.
I tried FCast with GrayJay and it worked perfectly, but I prefer Libretube over GrayJay so it would be great to have FCast in Libretube too.
